cmd/operator-sdk/add/api.go

```diff
@@ -20,90 +20,261 @@ import (
 	"os"
 	"path/filepath"
 
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	"k8s.io/client-go/discovery"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
 	"github.com/operator-framework/operator-sdk/cmd/operator-sdk/internal/genutil"
-	gencrd "github.com/operator-framework/operator-sdk/internal/generate/crd"
+	apiflags "github.com/operator-framework/operator-sdk/internal/flags/apiflags"
 	"github.com/operator-framework/operator-sdk/internal/scaffold"
+	"github.com/operator-framework/operator-sdk/internal/scaffold/ansible"
+	"github.com/operator-framework/operator-sdk/internal/scaffold/helm"
 	"github.com/operator-framework/operator-sdk/internal/scaffold/input"
 	"github.com/operator-framework/operator-sdk/internal/util/projutil"
-
-	log "github.com/sirupsen/logrus"
-	"github.com/spf13/cobra"
 )
 
-var (
-	apiVersion     string
-	kind           string
-	skipGeneration bool
-	crdVersion     string
-)
+var apiFlags apiflags.APIFlags
 
 func newAddAPICmd() *cobra.Command {
 	apiCmd := &cobra.Command{
 		Use:   "api",
 		Short: "Adds a new api definition under pkg/apis",
-		Long: `operator-sdk add api --kind=<kind> --api-version=<group/version> creates
-the api definition for a new custom resource under pkg/apis. This command
-must be run from the project root directory. If the api already exists at
-pkg/apis/<group>/<version> then the command will not overwrite and return
-an error.
-
-By default, this command runs Kubernetes deepcopy and CRD generators on
-tagged types in all paths under pkg/apis. Go code is generated under
-pkg/apis/<group>/<version>/zz_generated.deepcopy.go. CRD's are generated,
-or updated if they exist for a particular group + version + kind, under
+		Long: `operator-sdk add api --kind=<kind> --api-version<group/version> 
```

Comment:
The ansible new api is scaffolding a file called .placeolder inside of the dirs file and templates when should not. 

<img width="203" alt="Screenshot 2020-04-09 at 19 01 30" src="https://user-images.githubusercontent.com/7708031/78926178-8b5b8200-7a94-11ea-98a2-bd510c3125ae.png">
<img width="241" alt="Screenshot 2020-04-09 at 19 01 27" src="https://user-images.githubusercontent.com/7708031/78926182-8bf41880-7a94-11ea-9210-69d84724ee1c.png">

Reply:
Strange, I do not see that in my editor. 
@fabianvf Could you test as well, when you get a chance.

Reply:
you do not need any editor/idea. You can just run "ls -la" in the directory then you will see the file.